First lab

cseydlitz opened the pull request:
@anthonybotello Please Review. 
I am skeptical about my answers for Problem 3 parts C, D, and E
Also if you have any pointers on how to better explain parts D and E of the final question, I am open to suggestions.

cseydlitz committed:
Adding WIP lab1. Got to #2, successful Rayleigh dist.

cseydlitz committed:
Progress push

cseydlitz committed:
Initial Commit Lab 1

cseydlitz committed:
Update README.md

cseydlitz committed:
Update README.md

cseydlitz committed:
Delete lab1.py

No longer needed; code copied into workbook

cseydlitz committed:
Delete lab1.cpython-36.pyc

cseydlitz committed:
New upload of LAB1

Better graphs for Poisson distro

anthonybotello commented:
I'm not sure why you're importing everything in each individual cell. There isn't anything necessarily wrong with it, clearly, as you were able to complete the assignment. However, stylistically, I think it's more concise to have all of your imports at the beginning of the notebook.

My only real issue is with your explanation for part 1.D. You haven't addressed why you would get a negative number or what it would mean experimentally.

cseydlitz committed:
Final Upload

Changes made per @anthonybotello suggestion. 
All imports are now at the beginning of the notebook
Better explanation for negative sigma values in Q1 PartD

cseydlitz commented:
@anthonybotello Thank you for the feedback, changes made as suggested. Good to know about the imports; I didn't know you could just have one cell for all imports. 
Final workbook is up.

@oliviatessa Please review

oliviatessa commented on LAB1.html:
```diff
@@ -0,0 +1,13543 @@
+.btn-info.disabled.focus,
+.btn-info[disabled].focus,
+fieldset[disabled] .btn-info.focus {
+  background-color: #5bc0de;
+  border-color: #46b8da;
+}
+.btn-info .badge {
+  color: #5bc0de;
+  background-color: #fff;
+}
+.btn-warning {
+  co
```
In future labs, I don't think it is necessary to explain the methods you create in python. I'd rather you spend more time discussing the statistical concepts: probability, sigma, etcs.

oliviatessa replied:
Explain what the normal distribution is, what parameters it depends on, and the meaning of sigma.

oliviatessa replied:
Make sure you interpret your results and discuss whether they make sense.

oliviatessa replied:
Don't forget important details in your statistical question such as the nature of your background distribution.

oliviatessa replied:
This wouldn't be the sigma value that answers your statistical question. Sigma is meaningless for the Rayleigh distribution. Sigma is the value for the standard normal distribution that corresponds to your calculated probability.

oliviatessa replied:
Interesting, why is it appropriate to ask this question? Think about the difference between continuous and discrete distributions.

oliviatessa replied:
Can the mean of a Poisson distribution be non integer? What constraints does a discrete distribution put on sigma and probabilities?

oliviatessa commented:
Overall, I think your writeup could use more writing, particularly for the basic concepts explored at the beginning of the lab. Also, I think you need to revisit the concept of "sigma" as a probability. 

Some of your plots could be reformatted to be more clear. In particular, your Poisson plots were hard to follow.
